ledfx/effects/texter2d.py

```
@@ -260,9 +260,21 @@ def side_scroll_init(self):
         # first we need the first word width in screen units
 
-        offset = 1
         self.base_speed = self.speed_option_1
 
+        if self.base_speed != 0:
+            offset = 1
+        else:
+            # we want to center the text, so we need to know the overall sentence size
+            # we can then calculate the offset to center the text
+            sentence_width = 0
+            for idx, word in enumerate(self.sentence.wordblocks):
```

Ruff

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)